models/models/search.py

```diff
@@ -33,9 +33,16 @@ class SampleSearchResponseData(SearchResponseData):
     sample_external_ids: list[str]
 
 
+class ErrorSearchResponseData(SearchResponseData):
```

Comment:
Why subclass SearchResponesData, does the error belong to a project?
I would've expected just a straight:
```suggestion
class ErrorResponse(SMBase):
```